prow/cmd/tackle/main.go

```diff
@@ -318,7 +318,7 @@ func createCluster(proj, choice string) (*cluster, error) {
 		return nil, fmt.Errorf("select current zone for cluster: %v", err)
 	}
 
-	cmd := exec.Command("gcloud", "container", "clusters", "create", choice, "--zone="+zone)
+	cmd := exec.Command("gcloud", "container", "clusters", "create", choice, "--zone="+zone, "--enable-legacy-authorization", "--issue-client-certificate")
```

Comment:
How about when the cluster already exists?

Reply:
I can add a change to `mkbuild-cluster` to create a server certificate if it does not exist. Additionally, if legacy authorization is disabled, we can either *disable* it or add the appropriate `clusterbindings` to the `"client"` to make this work. I will implement this in a separate PR.